Intersect.Editor/Forms/DockingElements/frmMapEditor.cs

```diff
@@ -40,13 +42,41 @@ public partial class FrmMapEditor : DockContent
         private SwapChainRenderTarget mChain;
 
         private bool mMapChanged;
+        
+        // MapGrid Cursor
+        private Bitmap mCurSprite;
+
+        private string mCurPath;
+
+        private int mCurXClickPoint;
+        
+        private int mCurYClickPoint;
+
+        public struct IconInfo
+        {
+            public bool FIcon;
+            public int XHotspot;
+            public int YHotspot;
+            public IntPtr HbmMask;
+            public IntPtr HbmColor;
+        }
+
+        [DllImport("user32.dll")]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        private static extern bool GetIconInfo(IntPtr hIcon, ref IconInfo pIconInfo);
+
+        [DllImport("user32.dll")]
+        private static extern IntPtr CreateIconIndirect(ref IconInfo icon);
+
+        [DllImport("user32.dll")]
+        private static extern bool DestroyIcon(IntPtr hIcon);
 
         //Init/Form Functions
         public FrmMapEditor()
         {
             InitializeComponent();
-            Icon = System.Drawing.Icon.ExtractAssociatedIcon(System.Reflection.Assembly.GetExecutingAssembly().Location);
-            picMap.MouseLeave += (_sender, _args) => tooltipMapAttribute?.Hide();
+            Icon = Icon.ExtractAssociatedIcon(System.Reflection.Assembly.GetExecutingAssembly().Location);
+            picMap.MouseLeave += (sender, args) => tooltipMapAttribute?.Hide();
```

Comment:
If you're going to change the parameter names, change them both to `_` -- the underscore was to denote them not being used.